src/cljs/aikakonematka/core.cljs

```diff
@@ -95,15 +95,14 @@
     (println "puzzle-image-height : " @puzzle-image-height)
     (println "puzzle-width-height(* 0.7) : " (:puzzle-width-height @util/game-state))
     (println :game-state @util/game-state)
-    (let [actionOnClick (fn []
-                          (println "play button clicked"))
-          play-button (this-as this
+    (let [play-button (this-as this
                         (.button
                           game-object-factory
                           10
                           10
                           "play-button"
-                          actionOnClick
+                          (fn []
```

Comment:
Using anonymous function as a callback Because I think the function is simple and didn't deserve its own name and it's used only in one place